packages/paste-core/components/form/stories/form.stories.tsx

```diff
@@ -65,8 +240,180 @@ export const All = (): React.ReactNode => {
       <Box marginBottom="space70">
         <FormLabel htmlFor={inputIDSeed('errorinput')}>Text Input</FormLabel>
         <FormInput hasError id={inputIDSeed('errorinput')} type="text" value="" />
+        <Grid>
+          <Column>
+            <CheckboxGroup
```

Comment:
Seems like a lot of copy paste in this story. Can we component-ize this (like you did in inline-control-group) so the difference is easy to see?

Reply:
Probably good to give it a try